src/services/axios.ts

```diff
@@ -32,30 +33,23 @@ apiClient.interceptors.response.use(
     if (error.response && error.response.data) {
       const status = error.response.status;
       switch (status) {
-        case 400:
-          console.error("Bad Request: ", error.response.data.message);
+        case ErrorStatus.BAD_REQUEST:
+          window.history.back();
           throw new Error(error.response.data.message);
-        case 401:
-          console.error("Unauthorized: ", error.response.data.message);
+        case ErrorStatus.UNAUTHORIZED:
           Cookies.remove("token");
+          window.location.href = "/login";
           throw new Error(error.response.data.message);
-        case 404:
-          console.error("Not Found: ", error.response.data.message);
+        case ErrorStatus.NOT_FOUND:
+          window.location.href = "/not_found";
           throw new Error(error.response.data.message);
-        case 500:
-          console.error("Server Error: ", error.response.data.message);
-          Cookies.remove("token");
+        case ErrorStatus.SERVER_ERROR:
           throw new Error(error.response.data.message);
```

Comment:
ko cần throw => return server error page

Reply:
Này cũng sang trang Not Found ha anh

Reply:
page 500 chứ